doc/guidance.md

```diff
@@ -17,6 +17,7 @@ This is about how to use Doxygen to maintain the API documents. There are three
 	- cd $ROOT_AFR_DIR
 - Run generate_doc.sh to generate the documents. All warnings will go to $ROOT_AFR_DIR/doxygen_warnings.txt. 
   ```
+  $ chmod +x ./doc/generate_doc.sh
```

Comment:
Why not just change `generate_doc.sh` [to be executable instead?](https://www.jerriepelser.com/blog/execute-permissions-with-git/)

Reply:
Good point!